podcasts/ProfileViewController.swift

```diff
@@ -296,7 +300,7 @@ class ProfileViewController: PCViewController, UITableViewDataSource, UITableVie
 
     func tableView(_ tableView: UITableView, shouldHighlightRowAt indexPath: IndexPath) -> Bool {
         let row = tableData[indexPath.section][indexPath.row]
-        if FeatureFlag.kidsProfile.enabled, row == .kidsProfile {
+        if FeatureFlag.kidsProfile.enabled, row == .kidsProfile, !shouldHideKidsProfileBanner {
```

Comment:
Correct me if I'm wrong, but I believe we add the `kidsProfile` only if the flag is enabled and `shouldHideKidsProfileBanner` is `false`.

Checking the flag and the visibility here it's redundant, no? I removed both the feature flag and the visibility check and everything works as expected.

Reply:
My idea was to don't rely only on the datasource and be 200% sure it won't do anything.